self-core-impl/src/test/java/com/selfxdsd/core/ConditionalJsonResourcesTestCase.java

```diff
@@ -232,62 +251,24 @@ public void shouldGetFromRemoteIfRemoteHasChanged(){
         MatcherAssert.assertThat(result.toString(), Matchers.equalTo(
             newBody.toString()
         ));
-        
-        Mockito.verify(storage).store(uri, "etag-124", newBody.toString());
-    }
 
-    /**
-     * Should get Resource from remote if etag is present but 
-     * resource is missing.
-     */
-    @Test
-    public void shouldGetFromRemoteIfResourceEntryIsMissing(){
```

Comment:
@amihaiemil This test is not needed anymore. since we get the conditional resource in one go.